std/range/package.d

```diff
@@ -4173,10 +4195,10 @@ private string lockstepMixin(Ranges...)(bool withIndex)
             }
             return res;
         }
-    }, params.join(", "), withIndex ? "size_t index = 0;" : "",
+    }, name, params.join(", "), indexDef,
        emptyChecks.join(" && "), dgArgs.join(", "),
        popFronts.join("\n                "),
-       withIndex ? "index++;" : "");
```

Comment:
Hm. And we have to require StoppingPolicy.requireSameLength. Why is _stoppingPolicy a runtime value anyway? I guess it does add more possibilities, but is it really ever going to be used in practice? Ah well.